server/db/models/order-details.js

```diff
@@ -0,0 +1,25 @@
+// define OrderDetails model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
+
+const { UUID, UUIDV4, DECIMAL, INTEGER } = Sequelize;
+
+const OrderDetails = db.define('orderDetails', {
+    orderDetailsId: {
+        //how to define the orderDetailsId as concat(orderId, productId)?
```

Comment:
While making associations you can create aliases for the keys like orderIdProductId using 'as'. Is this what you mean?

Reply:
Not quite. I think "as" only specifies the column name. I wanted to make the system create an orderDetailId based on given orderId and productId. It seems complicated. So I just made it create its own unique UUID for now.

Reply:
Hm. Maybe order details will have two foreign keys (one that links to the orders and one that links to the products). I don't think concatting them will give us anything useful. Order details Id should be a UUID anyway tho.